examples/nbody/nbody.cpp

```diff
@@ -290,6 +293,8 @@ struct UpdateKernel
 #endif // NBODY_USE_SHARED
                         ts
                     );
+
+            syncBlockThreads(acc);
         };
```

Comment:
while you are at it, please drop the ';' here

Reply:
```suggestion
        }
```